backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/codec/impl/ElasticsearchLocalDateTimeFieldCodec.java

```diff
@@ -19,4 +20,9 @@ public ElasticsearchLocalDateTimeFieldCodec(DateTimeFormatter delegate) {
 	protected LocalDateTime nullUnsafeParse(String stringValue) {
 		return LocalDateTime.parse( stringValue, formatter );
 	}
+
+	@Override
+	protected Long nullUnsafeScalar(LocalDateTime value) {
+		return value.atZone( ZoneOffset.UTC ).toInstant().toEpochMilli();
```

Comment:
You can just call `value.toInstant( ZoneOffset.UTC )` instead of `atZone(...).toInstant()`.